tests/api.c

```diff
@@ -18989,6 +19049,7 @@ void ApiTest(void)
     test_wc_PKCS7_VerifySignedData();
     test_wc_PKCS7_EncodeDecodeEnvelopedData();
     test_wc_PKCS7_EncodeEncryptedData();
+     
```

Comment:
Please remove this blank line that was added, thanks.

Reply:
thanks